rules/windows/file/file_event/file_event_win_webshell_creation_detect.yml

```diff
@@ -1,42 +1,51 @@
-title: Windows Webshell Creation
+title: Potential Webshell Creation On Static Website
 id: 39f1f9f2-9636-45de-98f6-a4046aa8e4b9
 status: test
-description: Possible webshell file creation on a static web site
+description: Detects the creation of files with certain extensions on a static web site. Can be indicative of potential upload of webshell.
```

Comment:
```suggestion
description: Detects the creation of files with certain extensions on a static web site. This can be indicative of potential uploads of a web shell.
```